### ISSUE TITLE
translatade version ? 

### ISSUE THREAD

**Issue by FrankOcean11** [LikeFace/FollowHand#1]
i uploaded the code to my pc and the servos are jerking but no control, would fix if i can understand whats going on in the leap side .im using 2 servos only on pin 9 & 10 so i substituted 2 of the servo attach in setup thinking that would do the trick but nothing, help please? graciaz

**Comment by IsNotFunnyCompany** [comment]
you must compile and run the file "Following / Follows / Follows.ino" within the Arduino through the IDE. After that the configuration is this: inch = pin 3 arduino, index = pin 5 to Arduino, middle finger = pin 6 to Arduino, annular = pin 9 of Arduino, little finger = 10 pin of Arduino and the position of the pin 11 arduino is used for the rotation of the hand.
to work you must have attacked the leap motion...

// CODE
...
void setup()
{  
   Serial.begin(9600);

   Spollice.attach(3);  
   Sindice.attach(5); 
   Smedio.attach(6); 
   Sanuce.attach(9); 
   Smignolo.attach(10); 
   Smano.attach(11);  
}

...

**Comment by FrankOcean11** [comment]
what do you mean "must have attacked the leap motion..." ?

**Comment by IsNotFunnyCompany** [comment]
Arduino reads the data that unity 3d sends out the serial processing from Leap Motion. So to run the Leap Motion must be attached to the PC .

**Comment by IsNotFunnyCompany** [comment]
https://www.leapmotion.com/

**Comment by FrankOcean11** [comment]
yeah i dont think you understood, WHAT IS IT about leap that i have to setup ? because as far as im concern leap been installed and working in my pc prior to me knowing about processing and or arduino so can you be a bit more clear about what am i to have up and running aside from arduino have uploaded your example code?

**Comment by IsNotFunnyCompany** [comment]
ah ok , I do not speak English , or so I write in Italian or will I always rate the inverse translation with Google xD .

If the leap motion works , open Unity 3d and open the FollowAnimator scene ( Scene / FollowAnimator.unity ) , flaw go and be able to control the robot. 
After that go to read the Arduino COM port ( example COM3 ) and go to the file ( Scripts / Follow / FollowSerial.cs ) (https://github.com/LikeFace/FollowHand/blob/master/FollowLeapMotion/Assets/Scripts/Follow/FollowSerial.cs) to change public static SerialPort SerialPort = new SerialPort ( " COM3 " , 9600 ) ; with COM port read by Arduino . 
Proving HandOfGod scene should move servomotors (Scene/HandOfGod.unity)

ok?.... i wait answer

**Comment by FrankOcean11** [comment]
ok so there is the problem, it was never clear to me that i had to download and use Unity, thats ok this project is not for me then thank you for your time..... i just dont find a good tutorial on how to use JUST arduino and leap motion  or + processing
